src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,60 @@
-export const Pagination = () => {};
+// Pagination.tsx
+import React from 'react';
+
+interface PaginationProps {
+  pages: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  pages,
+  currentPage,
+  setCurrentPage,
+}) => {
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => setCurrentPage(currentPage > 1 ? currentPage - 1 : 1)}
+        >
+          «
+        </a>
+      </li>
+
+      {Array.from({ length: pages }).map((_, index) => (
+        <li
+          key={index}
+          className={`page-item ${currentPage === index + 1 ? 'active' : ''}`}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${index + 1}`}
+            onClick={() => setCurrentPage(index + 1)}
```

Comment:
Same as above, consider using `button` elements instead of `a` elements, or prevent the default action in the `onClick` handler to avoid potential issues with the hash in the URL.